website/integrations/services/pgadmin/index.md

```diff
@@ -13,89 +13,82 @@ sidebar_label: pgAdmin
 >
 > -- https://www.pgadmin.org/
 
-:::note
-This is based on authentik 2022.3.3 and pgAdmin4 6.19
-:::
-
 ## Preparation
 
 The following placeholders are used in this guide:
 
 - `pgadmin.company` is the FQDN of pgAdmin.
 - `authentik.company` is the FQDN of authentik.
 
-### Step 1: Create authentik Provider
-
-In authentik, under _Providers_, create an _OAuth2/OpenID Provider_ with these settings:
-
-**Provider Settings**
-
-- Name: pgAdmin
-- Client ID: Copy and Save this for Later
-- Client Secret: Copy and Save this for later
-- Redirect URIs/Origins: `http://pgadmin.company/oauth2/authorize`
-- Signing Key: Select any available key
-
-### Step 2: Create authentik Application
-
-In authentik, create an application which uses this provider. Optionally apply access restrictions to the application using policy bindings.
-
-- Name: pgAdmin
-- Slug: pgadmin
-- Provider: pgAdmin
-- Launch URL: https://pgadmin.company
-
-### Step 3: Configure pgAdmin
-
-All settings for OAuth in pgAdmin are configured in the `config_local.py` file. This file can usually be found in the path `/pgadmin4/config_local.py`
-
-:::note
-More information on that file can be found in the official pgAdmin [documentation](https://www.pgadmin.org/docs/pgadmin4/development/config_py.html)
-:::
-
-Copy the following code into the `config_local.py` file and replace all placeholders and FQDN placeholders
-:::note
-If the `config_local.py` file does not exist, it needs to be created in the `/pgadmin4/` directory.
-:::
-
-```py
-AUTHENTICATION_SOURCES = ['oauth2', 'internal']
-OAUTH2_AUTO_CREATE_USER = True
-OAUTH2_CONFIG = [{
-	'OAUTH2_NAME' : 'authentik',
-	'OAUTH2_DISPLAY_NAME' : '<display-name>',
-	'OAUTH2_CLIENT_ID' : '<client-id>',
-	'OAUTH2_CLIENT_SECRET' : '<client-secret>',
-	'OAUTH2_TOKEN_URL' : 'https://authentik.company/application/o/token/',
-	'OAUTH2_AUTHORIZATION_URL' : 'https://authentik.company/application/o/authorize/',
-	'OAUTH2_API_BASE_URL' : 'https://authentik.company/',
-	'OAUTH2_USERINFO_ENDPOINT' : 'https://authentik.company/application/o/userinfo/',
-	'OAUTH2_SERVER_METADATA_URL' : 'https://authentik.company/application/o/<app-slug>/.well-known/openid-configuration',
-	'OAUTH2_SCOPE' : 'openid email profile',
-	'OAUTH2_ICON' : '<fontawesome-icon>',
-	'OAUTH2_BUTTON_COLOR' : '<button-color>'
-}]
-```
+# authentik configuration
+
+1. From the Admin interface, navigate to **Applications** -> **Applications**.
+2. Use the wizard to create a new application and provider. During this process:
+    - Note the **Client ID**, **Client Secret**, and **slug** values because they will be required later.
+    - Set the redirect URI to `https://pgadmin.company`.
+    - Select any available signing key.
+
+## pgAdmin OAuth Configuration
+
+To configure OAuth in pgAdmin, you can either use the `config_local.py` file or set environment variables if you are deploying pgAdmin in a containerized setup.
```

Comment:
Do we ned to tell them exactly where to add the environment variables, as in which file? If we can, definitely let's do. Can be quite frustrating for users to be told to do something, but not where to do it.

Reply:
oh, I see you have it right below... ok.